scripts/syncup-s3.sh

```diff
@@ -40,6 +40,7 @@ sync_dirs=(
   pathway-analysis/data/leukemia
   pathway-analysis/data/medulloblastoma
   pathway-analysis/data/open-pbta
+  scRNA-seq-advanced/data/rms/processed
```

Comment:
Did the version you ran include `--acl public-read` below, or was that deleting that not needed? We should probably delete it anyway.

Reply:
I removed this line from the script, but didn't commit that change. It was definitely need it. I can remove it formally.